lib/provider/camunda/element-templates/CreateHelper.js

```diff
@@ -174,6 +176,35 @@ function createCamundaExecutionListenerScript(binding, value, bpmnFactory) {
 
 module.exports.createCamundaExecutionListenerScript = createCamundaExecutionListenerScript;
 
+/**
+ * Create camunda:Field element containing string or expression from given binding.
+ *
+ * @param {PropertyBinding} binding
+ * @param {String} value
+ * @param {BpmnFactory} bpmnFactory
+ *
+ * @return {ModdleElement}
+ */
+function createCamundaFieldInjection(binding, value, bpmnFactory) {
+  var DEFAULT_PROPS = {
+    'string': undefined,
+    'expression': undefined,
+    'name': undefined
+  };
+  var CAMUNDA_FIELD_EXTENSION_ELEMENT = 'camunda:Field';
```

Comment:
Please inline the variables directly and don't CAPSLOCK them. We're generally only using ALL_CAPS, if it is a global constant.

Reply:
Please rename `camunda:Field` to `camunda:field`. This is in line with the names of other extension points (`camunda:executionListener`, ...).